src/services/flows/service/flows.ts

```diff
@@ -145,3 +145,34 @@ export const validate = function (
     input_schema?: Record<string, any>;
   };
 }>;
+
+/**
+ * Deploy (Create) a Flow
+ * @see https://globusonline.github.io/globus-flows/#tag/Flows/paths/~1flows/post
+ */
+export const deploy = function (options?, sdkOptions?) {
```

Comment:
Just for consistency, since this is just a CRUD method, name it according to https://globus.github.io/globus-sdk-javascript/modules/_globus_sdk.html#md:service-method-names, but then maybe alias to `deploy` to better match the current documentation organization.

```suggestion
export const create = function (options?, sdkOptions?) {
```

Then...

```ts
export const deploy = create;
```